install/installer/pkg/components/spicedb/data/schema.yaml

```diff
@@ -6,12 +6,11 @@ schema: |-
   definition user {}
 
   definition organization {
-    // Organization specific roles
-    relation owner: user
+    // Every user in an organization is automatically a member
     relation member: user
-
-    // Membership is the union of all organization specific roles
-    permission membership = owner + member
```

Comment:
I think membership should not be a permission. I could potentially imagine that there are use-cases that this is an additional relation (think a role that has even less permissions than `member` and `owner`), but I don't see this now. :+1: